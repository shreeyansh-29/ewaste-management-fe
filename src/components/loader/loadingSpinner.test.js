/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import LoadingSpinner from "./loadingSpinner";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

describe("testing loader", () => {
  const wrapper = shallow(<LoadingSpinner />);

  it("test", () => {
    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
