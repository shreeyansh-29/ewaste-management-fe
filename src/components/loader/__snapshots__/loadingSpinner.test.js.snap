// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing loader test 1`] = `
<div
  className="spinner-container"
>
  <div
    className="loading-spinner"
  >
    <ThreeDots
      height="50"
      width="50"
    />
  </div>
</div>
`;
